grpc/src/main/java/com/linecorp/armeria/server/grpc/HandlerRegistry.java

```diff
@@ -267,12 +295,19 @@ static final class Entry {
         @Nullable
         private final Class<?> type;
 
+        @Nullable
+        private final Iterable<? extends Function<? super HttpService, ? extends HttpService>>
+                additionalDecorators;
```

Comment:
How about making `additionalDecorators` not null and passing an empty list if unspecified that can also handle the case where the builder methods pass empty decorators.